packages/nx/src/executors/run-commands/run-commands.impl.ts

```diff
@@ -169,14 +168,20 @@ function normalizeOptions(
       typeof c === 'string' ? { command: c } : c
     );
   }
+
+  if (options.args && typeof options.args === 'object') {
```

Comment:
```suggestion
  if (options.args && Array.isArray(options.args)) {
```

Reply:
Could you make this small change and then merge it?

Reply:
changed